src/test/java/org/springframework/data/elasticsearch/core/index/IndexOperationTransportTests.java

```diff
@@ -0,0 +1,53 @@
+package org.springframework.data.elasticsearch.core.index;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+
+import org.json.JSONException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.elasticsearch.core.ElasticsearchOperations;
+import org.springframework.data.elasticsearch.core.IndexOperations;
+import org.springframework.data.elasticsearch.core.mapping.IndexInformation;
+import org.springframework.data.elasticsearch.junit.jupiter.ElasticsearchTemplateConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+
+/**
+ * @author George Popides
+ */
+
+@ContextConfiguration(classes = { ElasticsearchTemplateConfiguration.class })
+public class IndexOperationTransportTests extends IndexOperationTests {
+	@Autowired
+	protected ElasticsearchOperations operations;
+
+	@Test // #1646
+	@DisplayName("should return info of all indices using TRANSPORT template with no aliases")
+	public void shouldReturnInformationListOfAllIndicesNoAliases() throws JSONException {
```

Comment:
this method is not needed. It does the same as the one in the base class. The purpose of the `IndexOperationTransportTests` class is to run exactly the same tests as in `IndexOperationTests` just having a different implementation of `ElasticsearchOperations`.